app/static/js/volunteerProfile.js

```diff
@@ -125,27 +125,40 @@ $(document).ready(function(){
         }
     });
 
-  $("#updatePhone").click(function() {
+  $("#updatePhone").on('click', function() {
     userName = $(this).data("username")
-    phoneNumber = $("#phoneInput").val()
-    let isvalid = phoneNumber.replace(/\D/g,"").length === 10;
+    if ($(this).html() === 'Edit') {
+        $(this).html('Save')
+        $("#phoneInput").focus()
+    } else {
+        // Save the phone number
+        var phoneInput = $("#phoneInput");
+        let isvalid = phoneInput.val().replace(/\D/g,"").length === 10;
+        let isempty = phoneInput.val().replace(/\D/g,"").length === 0;
+        if (!(isvalid || isempty)) {
+            phoneInput.addClass("invalid");
+            window.setTimeout(() => phoneInput.removeClass("invalid"), 1000);
+            phoneInput.focus()
+            return false;
+        }
+        $(this).html('Edit');
```

Comment:
Phone number handling is now duplicated. Let's make a function that these two places can share.